src/main/java/ui/UiRenderer.java

```diff
@@ -50,6 +54,7 @@ public static void printWeekBody(LocalDate startOfWeek, DateTimeFormatter dateFo
     }
 
     private static void printDateRow(DateTimeFormatter dateFormatter, LocalDate date) {
+        logger.log(Level.INFO, "Printing dates for week starting from " + date);
```

Comment:
This is a good start for logging in a log file instead of printing to a console! 👍